modules/integration/src/test/scala/scala/cli/integration/InstallAndUninstallCompletionsTests.scala

```diff
@@ -0,0 +1,49 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+class InstallAndUninstallCompletionsTests extends munit.FunSuite {
+  val zshRcFile  = ".zshrc"
+  val bashRcFile = ".bashrc"
+  val rcContent = s"""
+                      |dummy line
+                      |dummy line"""
+  val testInputs = TestInputs(
+    Seq(
+      os.rel / zshRcFile  -> rcContent,
+      os.rel / bashRcFile -> rcContent
+    )
+  )
+
+  def runInstallAndUninstallCompletions(): Unit = {
+    testInputs.fromRoot { root =>
+      val zshRcPath  = root / zshRcFile
+      val bashRcPath = root / bashRcFile
+      // install completions to the dummy rc files
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", zshRcPath, "--shell", "zsh").call(
+        cwd = root
+      )
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", bashRcPath, "--shell", "bash").call(
+        cwd = root
+      )
+      val completionsStart = s"# >>> scala-cli completions >>>"
+      val completionsEnd   = s"# <<< scala-cli completions <<<"
```

Comment:
No need of string interpolation here:
```suggestion
      val completionsStart = "# >>> scala-cli completions >>>"
      val completionsEnd   = "# <<< scala-cli completions <<<"
```